index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio : Mobile version skeleton</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+  </head>
+  <body>
+    <!-- Navbar part   -->
+    
+    <header>
+      <nav>
+        <div class="nav-item brand">John Doe</div>
```

Comment:
- [ ] Kindly remove the logo from the nav tag only the hamburger icon should be in the nav tag, please change div tag to anchor tag because this is part of the project requirements👍🏿